database/migrations/2024_01_24_063519_job_feedback_options.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use App\Models\Extensions\BaseConfigMigration;
+
+return new class() extends BaseConfigMigration {
+	public const PROCESSING = 'Image Processing';
+	public const POSITIVE = 'positive';
```

Comment:
```suggestion
```

Not needed?

Reply:
yeah left over from another migration :laughing: